guides/common/modules/proc_enabling-the-flatpak-remote.adoc

```diff
@@ -65,11 +65,22 @@ $ hammer flatpak-remote remote-repository mirror \
 --id=_My_Remote_Repo_ID_ \
 --product-id=_{Project}_Product_ID_
 ----
-You can view the repository under the selected product in {ProjectWebUI}.
++
+[NOTE]
+====
+The `--id` in this command refers to the Flatpak remote repository ID, not the standard {Project} repository ID.
+====
+
+. You can view the repository under the selected product in {ProjectWebUI}.
```

Comment:
Is it intentional to make this a separate step? Before, it was a instruction just below the code block.